### DISCUSSION TITLE
What is an alternative to getBoundingBox in sceneview 2.0.4 which was present in sceneview 2.0.0

### DISCUSSION THREAD

**Discussion by Somendra121** [sceneview/sceneview#511] [Q&A]
What is an alternative to getBoundingBox in sceneview 2.0.4 which was present in sceneview 2.0.0

**Comment by Somendra121** [comment]
Please help

**Comment by ThomasGorisse** [comment]
In v4.0.0-rc.1, you can access the bounding box through the Filament `RenderableManager`:

```kotlin
val modelInstance = rememberModelInstance(modelLoader, "models/mymodel.glb")
if (modelInstance != null) {
    val boundingBox = modelInstance.asset.boundingBox
    // boundingBox gives you min/max extents of the model
}
```

The `sceneview-core` module also provides collision primitives (`Box`, `Sphere`) that can be used for hit testing and spatial queries.
